providers/github/types/index.json

```diff
@@ -1 +1 @@
-{"Resources":{"repositories@v1":{"RelativePath":"github/repositories/v1/types.json","Index":27}}, "Functions":{}}
+{"Types":{"repositories@v1":{"RelativePath":"github\\repositories\\v1\\types.json","Index":27},"repositories/collaborators@v1":{"RelativePath":"github\\repositories\\v1\\types.json","Index":35}}}
```

Comment:
You'll need to fix this to adhere to the new format

Reply:
Oops, forgot to update it after the second generation.